diff --git a/node_modules/@react-native-community/async-storage/android/build.gradle b/node_modules/@react-native-community/async-storage/android/build.gradle
index 4dc43dc..5057cb1 100644
--- a/node_modules/@react-native-community/async-storage/android/build.gradle
+++ b/node_modules/@react-native-community/async-storage/android/build.gradle
@@ -9,7 +9,7 @@ buildscript {
     if (project == rootProject) {
         repositories {
             google()
-            jcenter()
+            mavenCentral()
         }
         dependencies {
             classpath 'com.android.tools.build:gradle:3.5.0'
@@ -56,7 +56,7 @@ repositories {
         url "$rootDir/../node_modules/react-native/android"
     }
     google()
-    jcenter()
+    mavenCentral()
 }
 
 dependencies {
