diff --git a/node_modules/@react-native-community/viewpager/android/build.gradle b/node_modules/@react-native-community/viewpager/android/build.gradle
index 03ebdad..ec99827 100644
--- a/node_modules/@react-native-community/viewpager/android/build.gradle
+++ b/node_modules/@react-native-community/viewpager/android/build.gradle
@@ -1,7 +1,7 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
     }
 
     dependencies {
@@ -31,7 +31,6 @@ android {
 
 repositories {
     google()
-    jcenter()
     mavenCentral()
 }
 
