diff --git a/node_modules/react-native-picker-module/android/build.gradle b/node_modules/react-native-picker-module/android/build.gradle
index 5fe3d25..b858d4b 100644
--- a/node_modules/react-native-picker-module/android/build.gradle
+++ b/node_modules/react-native-picker-module/android/build.gradle
@@ -1,7 +1,7 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:3.3.0'
